ledger-tool/src/main.rs

```diff
@@ -188,6 +188,10 @@ fn output_slot(
     verbose_level: u64,
     all_program_ids: &mut HashMap<Pubkey, u64>,
 ) -> Result<(), String> {
+    for (key, value) in blockstore.dump(slot) {
```

Comment:
obviously, this isn't good to merge as-is.

Ideally, we should add new ledger-tool subcommand like this:

```
solana-ledger-tool rocksdb-dump --slot NNNN --column-family CF
```